pulp_ansible/app/viewsets.py

```diff
@@ -200,30 +198,33 @@ class AnsibleRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin):
         description="Trigger an asynchronous task to sync Ansible content.",
         responses={202: AsyncOperationResponseSerializer},
     )
-    @action(detail=True, methods=["post"], serializer_class=RepositorySyncURLSerializer)
+    @action(detail=True, methods=["post"], serializer_class=AnsibleRepositorySyncURLSerializer)
     def sync(self, request, pk):
         """
         Dispatches a sync task.
         """
         repository = self.get_object()
-        serializer = RepositorySyncURLSerializer(
+        serializer = AnsibleRepositorySyncURLSerializer(
             data=request.data, context={"request": request, "repository_pk": repository.pk}
         )
         serializer.is_valid(raise_exception=True)
 
         remote = serializer.validated_data.get("remote", repository.remote)
         remote = remote.cast()
 
+        mirror = serializer.validated_data.get("mirror", False)
```

Comment:
This is similar, I don't think we should have a default here if the serializer provides one.